cli/pkg/release/gbm.go

```diff
@@ -203,20 +206,124 @@ func CreateGbmPR(version, dir string) (gh.PullRequest, error) {
 }
 
 func renderGbmPrBody(version string, pr *gh.PullRequest) error {
+	// TODO - replace "" with dir variable
+	cl := getChangeLog("", pr)
+	rn := getReleaseNotes("", pr)
```

Comment:
In the [first version of this code](https://github.com/wordpress-mobile/release-toolkit-gutenberg-mobile/blob/gbm-cli/cli/pkg/release/gbm.go#L114-L115), `dir` was being passed to the `renderGbmBody` function. Adding a TODO to update it.